### DISCUSSION TITLE
Is the WebAPI hosted at a publicly accessible URL?

### DISCUSSION THREAD

**Discussion by NeilMcKechnie** [Green-Software-Foundation/carbon-aware-sdk#110] [General]
While it wouldn't be too hard to get this code running in our own environment, it would also be very convenient if it were already up and running somewhere...at a minimum for experimentation and prototyping.

I'm new to this project, so maybe it is?

**Comment by danuw** [comment]
Hi @vaughanknight is about to update with the latest update and will share the link here...
(this should happen in the next ~24 hours)

**Reply by vaughanknight** [reply]
Apologies, had some issues this week and will get to this today.  Will share the URL here and will update for the hackathon too.

**Comment by Willmish** [comment]
Publicly available hosted web API can be accessed here for the duration of the hackathon: https://grnsft.org/hack22/api

*NOTE: After the hackathon has ended, reach out to WattTime for keeping a full WattTime license for your application*
